app/src/main/AndroidManifest.xml

```diff
@@ -15,7 +15,8 @@
             android:name=".app.MainActivity"
             android:launchMode="singleTop"
             android:windowSoftInputMode="adjustPan"
-            android:label="@string/app_name">
+            android:label="@string/app_name"
+            android:exported="true">
```

Comment:
Any reason for making this exported? AFAIK nothing outside the app package needs to reference this activity